codewars/8kyu/arrayMap.js

```diff
@@ -0,0 +1,17 @@
+let numbers = [1, 3, 5, 7, 8, 9, 11];
+let numbersMaped = [];
+numbersMaped = numbers.map(function(el) {
+    return el * 2;
+})
+
```

Comment:
there should be no more than one line of space between code blocks

Reply:
Done